tests/engine/artifact_filters.py

```diff
@@ -186,7 +187,13 @@ def testBuildFindSpecsFromFileArtifact(self):
     self.assertEqual(len(find_specs), 1)
 
     # Location segments should be equivalent to \Windows\test_data\*.evtx.
-    path_segments = ['Windows', 'test\\_data', '.*\\.evtx']
+    if py2to3.PY_3:
```

Comment:
Shouldn't the check here be `Python version >= 3.3` ?

Reply:
We don't support any python version below 3.4, so that's not necessary.

Reply:
I understand but if change the check we technically can remove the comment.

Reply:
I don't think it's worth the extra complexity. Changed the docstring a bit.

Reply:
ack.